release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -730,6 +730,15 @@ This release introduces the following customizations for the descheduler:
 
 For more information, see xref:../nodes/scheduling/nodes-descheduler.adoc#nodes-descheduler[Evicting pods using the descheduler].
 
+[id="ocp-4-11-nodes-node-maintenance-operator"]
+==== Node Maintenance Operator enhancements
+The Node Maintenance Operator provides the following enhancements:
+
+* Additional feedback, `drainProgress` and `lastUpdate`, is now provided regarding the status of `NodeMaintenance` CR tasks.
+* For clusters with bare-metal nodes, an easier method is now available where you can place a node into maintenance mode, and resume a node from maintenance mode, by using the web console *Actions* control.
```

Comment:
maybe add somthing like ".. on the Compute -> Nodes site"?

Reply:
Thanks for you feedback @slintes I have updated it as agreed by including web console, and removing the specific actions.